src/main/webapp/assets/modules/common/totalFilter.js

```diff
@@ -0,0 +1,7 @@
+techlooper.filter("total", function (jsonValue, $filter) {
```

Comment:
'techlooper' is not defined.<br>'$filter' is defined but never used.<br>'jsonValue' is defined but never used.